doc/mgr/nfs.rst

```diff
@@ -242,13 +242,13 @@ permissible values.
 .. note:: Export creation is supported only for NFS Ganesha clusters deployed using nfs interface.
 
 Create RGW Export
-=================
+-----------------
 
 To export a bucket
 
 .. code::
 
-   $ ceph nfs export create rgw <bucket_name> <cluster_id> <pseudo_path> [--readonly] [--client_addr <value>...] [--realm <value>]
+   $ ceph nfs export create rgw <bucket_name> <cluster_id> <pseudo_path> [--readonly] [--client_addr <value>...] [--squash <value>]
```

Comment:
```suggestion
   $ ceph nfs export create rgw <bucket_name> <cluster_id> <pseudo_path> [--readonly] [--client_addr <value>...] [--realm <value>] [--squash <value>]
```

Reply:
I removed it because the --realm arg is a no-op; rgw exports only work on teh default realm due to the way the rgw fsal is implemented.  I added a patch that removes the argument from the code too.